bin/ansible-connection

```diff
@@ -291,9 +291,14 @@ def main():
         sys.stderr.write(traceback.format_exc())
         sys.exit("FAIL: %s" % e)
 
+    # disable logging on stdout, to avoid spurious debug logs to
+    # be outputted on stdout.
+    C.DEFAULT_LOG_ONLY = True
```

Comment:
Ok, I will close this PR and try to figure out some other ways to fix this issue. Thank you for the review.